lib/rack/request.rb

```diff
@@ -151,11 +157,11 @@ def request_method;  get_header(REQUEST_METHOD)                     end
       def query_string;    get_header(QUERY_STRING).to_s                  end
       def content_length;  get_header('CONTENT_LENGTH')                   end
```

Comment:
So I wanted to change this use of `'CONTENT_LENGTH'` and a couple uses of `'CONTENT_TYPE'`, but we have a conundrum.

There's a constant defined `CONTENT_LENGTH = 'Content-Length'.freeze`, notice the NOT ALL CAPS, so switching this to use the constant value is actually incorrect for this case.